src/lib/wallet.js

```diff
@@ -1,16 +1,14 @@
-import BrowserWindowMessageConnection from '@aeternity/aepp-sdk/es/utils/aepp-wallet-communication/connection/browser-window-message';
-import { isEqual, times } from 'lodash-es';
+import { isEqual } from 'lodash-es';
 import {
   NODE_STATUS_CONNECTING,
   NODE_STATUS_ERROR,
-  executeAndSetInterval,
-  watchUntilTruthy,
   NODE_STATUS_CONNECTED,
+  connectFrames,
```

Comment:
Is `connectFrames` defined somewhere?

Reply:
Yes, in the `utils.ts` file. But I'm thinking if it would make sense to create lib out of this. What do you think?

Reply:
Right, it was introduced in different PR, but the code here wasnt using extracted function.